web/ui/react-app/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+      "react-app",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "@typescript-eslint/camelcase": "warn",
+      "eol-last": [
+        "error",
+        "always"
+      ],
+      "prefer-const": "warn",
+      "comma-dangle": ["error", {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "always-multiline"
+    }]
+    },
+    "plugins": [
+      "prettier"
+    ]
+  }
```

Comment:
missing newline file termination here (not sure if we can have a linter setting for the linter config file itself :))